CMakeLists.txt

```diff
@@ -1,16 +1,122 @@
-cmake_minimum_required(VERSION 3.0)
-project(colorpick)
+cmake_minimum_required(VERSION 3.15)
+project(Colorpick
+    VERSION 0.1.0
+    LANGUAGES CXX
+)
+string(TOLOWER ${PROJECT_NAME} PROJECT_ID)
 
-set(APP_NAME ${PROJECT_NAME})
-
-find_package(Qt5 CONFIG REQUIRED Core Widgets)
-find_package(KF5GuiAddons CONFIG REQUIRED)
-find_package(KF5WidgetsAddons CONFIG REQUIRED)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Woverloaded-virtual")
+set(CMAKE_CXX_STANDARD 17) #Qt6 needs C++17
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+#===============================================================================
+# Qt
+#===============================================================================
```

Comment:
nit: I prefer not to have banner-style comments as I find them tedious to maintain. I prefer something simpler, like this (note the starting empty line):
```suggestion

#
# Qt
#
```